### PR TITLE
Update ANN.py

### PR DESCRIPTION
**Pull request by abkhaoula**
ok

### CODE REVIEW

File: loaded/Logistic normalised/ANN.py [hunk 1]
```diff
@@ -1,6 +1,4 @@
-# %%
 
-import warnings
 from tensorflow_core.python.keras.wrappers.scikit_learn import KerasClassifier
 from sklearn.model_selection import GridSearchCV
```
Comment (abkhaoula) [inline loaded/Logistic normalised/ANN.py:3]: nnnn

Comment (abkhaoula) [reply loaded/Logistic normalised/ANN.py:3]: ok
